apps/experiments/views/experiment.py

```diff
@@ -363,6 +363,19 @@ def extra_context(self):
             team_participant_identifiers.extend(self.object.participant_allowlist)
             team_participant_identifiers = set(team_participant_identifiers)
 
+        excluded_fields = [
```

Comment:
offlined, but outstanding questing if these can be added to the default excluded keys

Reply:
Seems fine to add `version_number`. I think the others are experiment specific so probably best to override `get_fields_to_exclude` on the Experiment model to add them there.